README.md

```diff
@@ -1,20 +1,29 @@
 ## Cloud SQL Connector for Java
 
-The Cloud SQL Connector for Java is a library for the MySQL/PostgreSQL JDBC and R2DBC drivers that allows a user 
-with the appropriate permissions to connect to a Cloud SQL database without having to deal with IP 
-whitelisting or SSL certificates manually.
+![CI Java 8][ci-badge-java-8]
+![CI Java 11][ci-badge-java-8]
 
+The Cloud SQL Connector for Java is a library that provides IAM-based authorization and encryption when connecting to a Cloud SQL instance. It can *not* provide a network path to a Cloud SQL instance if one is not already present.  See the [Connecting Overview](https://cloud.google.com/sql/docs/mysql/connect-overview) page for more information on connecting to a Cloud SQL instance.
 
+## Usage
+For usage information specific to your database engine and driver, see the pages below:
 
-## Examples
+JDBC:
+* [Connecting to MySQL using JDBC](docs/jdbc-mysql)
```

Comment:
.md?

Reply:
Is there a particular reason we're using the docs folder? Would it be better to put in the README for the artifact itself?

Reply:
This is for consistency with the documentation structure of the proxy repo. We can do artifact READMEs instead if you think that's better.